src/Card/SummaryCard/SummaryCard.tsx

```diff
@@ -212,39 +176,27 @@ function SummaryCard({
           }
           disableTypography
           title={
-            <div>
-              <Tooltip title={title} disableHoverListener={!titleSizeOverflow}>
-                <Typography
-                  ref={titleRef}
-                  sx={{
-                    fontSize: 20,
-                    fontWeight: 500,
-                    width: headerContentWidth
-                  }}
-                  noWrap
-                >
-                  {title}
-                </Typography>
-              </Tooltip>
-            </div>
-          }
-          subheader={
-            <Tooltip
-              title={subtitle}
-              disableHoverListener={!subTitleSizeOverflow}
-            >
+            <TruncatedTooltip tooltip={title}>
```

Comment:
If I have an overflowing title as well as an action button, the button gets pushed off the card.

![image](https://github.com/IPG-Automotive-UK/react-ui/assets/27866636/df0ea2e1-a3a4-45b5-b6a8-a0b1477cacd4)